source/events/issue_handler/actions/created/commands/go/source/breaking-changes/findBreakingChanges.ts

```diff
@@ -107,15 +113,31 @@ async function extractBreakingChanges(packageName: string, cursorVersion: string
 
     try {
         // here we need to specify function to have good JSON reply structure
-        const completion = await chatCompletion(breakingChangesConversation, MODEL);
-
+        const completion = await chatCompletion(breakingChangesConversation, GPT4_MODEL);
```

Comment:
With larger models like prisma this would crash - too long for GPT4_MODEL + our prompt. We need to use tokenizer to determine prompt + changelog length and if it's less than MAX_NUM_TOKENS_8K use GPT 4. Double check prompt should have enough space by default with GPT 4 since there will be lot less info.